lacework/resource_lacework_alert_channel_datadog.go

```diff
@@ -0,0 +1,254 @@
+package lacework
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/lacework/go-sdk/api"
+)
+
+func resourceLaceworkAlertChannelDatadog() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkAlertChannelDatadogCreate,
+		Read:   resourceLaceworkAlertChannelDatadogRead,
+		Update: resourceLaceworkAlertChannelDatadogUpdate,
+		Delete: resourceLaceworkAlertChannelDatadogDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkIntegration,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"intg_guid": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"datadog_site": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  string(api.DatadogSiteCom),
+				ValidateFunc: func(value interface{}, key string) ([]string, []error) {
+					switch value.(string) {
+					case string(api.DatadogSiteEu), string(api.DatadogSiteCom):
+						return nil, nil
+					default:
+						return nil, []error{
+							fmt.Errorf(
+								"%s: can only be either 'eu' or 'com'", key,
```

Comment:
What about this? 😃 
```suggestion
								"%s: can only be either '%s' or '%s'", key,
								api.DatadogSiteEu, api.DatadogSiteCom,
```

Reply:
Since we have such a nice enum's... Let us use them! Thoughts?

Reply:
I like it!